### ISSUE TITLE
Correctly handle slashes in branch names

### ISSUE THREAD

**Issue by solflare** [danielbeardsley/cimpler#77]
An incorrect copy/paste led to an url being used as the branch name being queued in cimpler. When cimpler attempted to run the branch the entire process died as it was unable to correctly handle the slashes present.

**Comment by danielbeardsley** [comment]
```
cimpler[6393]: [2017-07-28 14:57:28.581] [WARN] [default] - http://example.com/Misc/screenshots/SHNaV.png -- Fetch Failed
cimpler[6393]: events.js:141
cimpler[6393]:       throw er; // Unhandled 'error' event
cimpler[6393]:       ^
cimpler[6393]: Error: ENOENT: no such file or directory, open '/var/www/ifixit/pub/Misc/cimpler-logs/ifixit/http://example.com/Misc/screenshots/SHNaV.png--unknown--1501279048581.log'
cimpler[6393]:     at Error (native)
systemd[1]: cimpler.service: Main process exited, code=exited, status=1/FAILURE
systemd[1]: cimpler.service: Unit entered failed state.
systemd[1]: cimpler.service: Failed with result 'exit-code'.
systemd[1]: cimpler.service: Service hold-off time over, scheduling restart.
```

Perhaps we should alphanumeric-ize the branch name before using it in the log file name.

**Comment by danielbeardsley** [comment]
I'm almost certain this was solved by #84
